site/examples/spinner.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const spinner = `<div class=' viewport-full relative clip'>
+  <div class='flex-parent flex-parent--center-cross flex-parent--center-main bg-darken10 viewport-twothirds viewport-full-ml'>
```

Comment:
- I don't think `viewport-twothirds` & `viewport-full-ml` are necessary?
- The way this is set up right now, the fullscreen loading container would push the content down it should be overlaid on top of. I would give this container a these classes instead: `absolute top left right bottom` to fill the entire size and be positioned outside of the document flow. I would also add the highest z-index value we provide in Assembly `z5` to ensure it's above everything.